globals/prefixes.go

```diff
@@ -163,10 +163,24 @@ var prefixToResourceType = map[string]resource.Type{
 	SessionRecordingPrefix:                     resource.SessionRecording,
 }
 
+var resourceTypeToPrefixes map[resource.Type][]string = func() map[resource.Type][]string {
+	ret := make(map[resource.Type][]string)
+	for k, v := range prefixToResourceType {
+		ret[v] = append(ret[v], k)
+	}
+	return ret
+}()
```

Comment:
If this is the only use of this new global, maybe put in as a local variable in `ResourcePrefixesFromType` and use a global sync.Once to populate it? That'd make it immutable at runtime.

Reply:
If it's local how would you not have to populate it each time you call the function? You leave the function, it falls out of scope. Or do you mean keep the variable at the package level, but simply do the population in `ResourcePrefixesFromType`?

That does feel like it adds some complexity and I'm not at all worried about it being modified at runtime. 🤷

Reply:
You're right, it'd still have to be a global, nevermind me!